packages/dso-toolkit/src/components/list/list.scss

```diff
@@ -93,3 +93,33 @@ ol {
     @include list.unstyled();
   }
 }
+
+ul {
+  > li.forbidden,
```

Comment:
Graag alle classes prefixen met `dso-`.

Reply:
Daar twijfelde ik hierbij over.

Reply:
Inderdaad, hier voel ik met @LukeVerlaan mee; het is een soort ‘State’. Moeten die ook geprefixed worden?

Reply:
Alles prefixen. Het kan zomaar zijn dat er een andere library in de applicatie van de afnemer op `forbidden` heeft gestijld. De namespace werkt beide kanten op: We beinvloeden anderen niet maar laten ons ook niet door andere beinvloeden.